src/ethereum/prague/requests.py

```diff
@@ -8,20 +8,11 @@
 [EIP-7685]: https://eips.ethereum.org/EIPS/eip-7685
 """
 
-from dataclasses import dataclass
-from typing import Tuple, Union
+from hashlib import sha256
```

Comment:
Should we create a wrapper in `ethereum.crypto.hash` like we do for `keccak256`?

Reply:
Unless we plan on moving this whole function (for computing the hash of a list of bytes) to `ethereum.crypto`, we will need it here